docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -1173,6 +1186,362 @@ SQL> POPULATE INDEX index_target_table4 ON target_table4;
 SQL> DROP INDEX index_target_table4;
 --- SQL operation complete.
 ```
+
+[[rebuild_indexes_examples]]
+==== Examples of `REBUILD INDEXES`
+
+Suppose that we have following tables:
+
+_source_table_:
+
+```
+SQL>select count(*) from source_table;
+(EXPR)
+--------------------
+             1000000
+
+--- 1 row(s) selected. 
+```
+
+_target_table1_ has the same structure as _target_table2_, here takes _target_table1_ for example:

```

Comment:
Possible wordsmith: "Suppose _target_table1_ and _target_table2_ both have the following structure:"

Reply:
Thanks Dave, your comment has been incorporated :)